auditbeat/module/file_integrity/monitor/recursive.go

```diff
@@ -154,7 +161,7 @@ func (watcher *recursiveWatcher) forwardEvents() error {
 			switch event.Op {
 			case fsnotify.Create:
 				err := watcher.addRecursive(event.Name)
-				if err != nil {
+				if err != nil && !watcher.isExcludedPath(event.Name) {
```

Comment:
So the changes to this function are ignoring errors related to ignored paths. Does it still make sense to act on events for ignored paths?

For example, adding a recursive watch on an ignored directory. I wonder if it will make more sense to just skip events for ignored paths.

Reply:
@adriansr Thanks for the review, it makes more sense to stop watching them. I've updated this PR regarding your suggestion, please have a look.